### ISSUE TITLE
image gallery and left/right arrows and esc key

### ISSUE THREAD

**Issue by crutkas** [Microsoft/WindowsTemplateStudio#2254]
### Repro steps

Click image
use arrows to nav
use esc to go back to main screen

### Expected Behavior

- I click 9, would expect it to go to 8 if i used the left arrow, 10 for right
- I click esc, i would go back to the larger grid

### Actual Behavior

- nothing happens, until i click image for arrow support
- Esc key does nothing

## System

* **VS Version:** 15.7
  ```
<genTemplate:Metadata>
    <genTemplate:Item Name="generator" Value="Windows Template Studio"/>
    <genTemplate:Item Name="wizardVersion" Version="v0.17.18095.1" />
    <genTemplate:Item Name="templatesVersion" Version="v0.17.18108.1" />
    <genTemplate:Item Name="projectType" Value="SplitView" />
    <genTemplate:Item Name="framework" Value="CodeBehind" />
  </genTemplate:Metadata>
```

**Comment by mvegaca** [comment]
Fixes on CSharp and VB templates on PR #2259

**Comment by mrlacey** [comment]
This makes the escape key act like the back button.

The escape key should be used for dismissing transient UI but when viewing an image from the gallery a page navigation has taken place. It should not be used for page navigation. Doing this introduces an inconsistency in the generated code and goes against design guidelines.

[Design docs on keyboard usage.](https://docs.microsoft.com/en-us/windows/uwp/design/input/keyboard-interactions)



**The original request for this issue is wrong. The ESc key should not trigger backward navigation!**

**Comment by crutkas** [comment]
how the UX is laid out for this page, it should act like the photos app.  The photos app if you hit esc, it brings you back to the grid.

**Comment by mrlacey** [comment]
> how the UX is laid out for this page, it should act like the photos app. The photos app if you hit esc, it brings you back to the grid.

then the photos app isn't following the platform guidance.

**Comment by sibille** [comment]
Verified in dev-nightly:
Templates version: 0.18.18149.2
Wizard version: 0.18.18149.1
